helix-term/src/commands.rs

```diff
@@ -6210,3 +6211,33 @@ fn replay_macro(cx: &mut Context) {
         },
     ));
 }
+
+fn show_subtree(cx: &mut Context) {
+    let (view, doc) = current!(cx.editor);
+
+    if let Some(syntax) = doc.syntax() {
+        let primary_selection = doc.selection(view.id).primary();
+        let text = doc.text().slice(..);
```

Comment:
Do you need to take a slice? I think it should still work with just `doc.text()`

Reply:
Right you are, thanks!